src/main/groovy/org/octopusden/octopus/license/management/plugins/gradle/LicenseGradlePlugin.groovy

```diff
@@ -116,7 +116,9 @@ class LicenseGradlePlugin implements Plugin<Project> {
             onlyCurrentProject = onlyCurrent
         }
         Task processLicenses = project.getTasks().create(processLicensesTaskName, LicenseTask.class)
+        def isLicenseSkipFalse = propertyIsFalse(project, LICENSE_SKIP_PROPERTY)
```

Comment:
Double negation can be misleading; it’s better to rename the parameter, for example, to isLicenseRequired